.gitlab/llnl-ci-conda.yml

```diff
@@ -7,127 +7,65 @@ stages:
   - setup
   - build
   - test
-  - cleanup
+
+.on_quartz:
+  variables:
+    HOST: "quartz"
+    CONDA_ENV_NAME: "quartz-env"
+    CONDA_ARCH: "x86_64"
+
+.on_ruby:
+  variables:
+    HOST: "ruby"
+    CONDA_ENV_NAME: "ruby-env"
+    CONDA_ARCH: "x86_64"
+
+.job_tags:
+  tags:
+    - $HOST
+    - $RUNNER_TYPE
 
 .final_steps:
   script: &finalize
     - chgrp -fR exaworks ${CONDA_WORK_DIR} || true
     - chmod -fR 02770 ${CONDA_WORK_DIR}    || true
 
-conda-setup:
+.conda_setup:
   stage: setup
-  parallel:
-    matrix:
-      - HOST : [ "quartz" ]
-        CONDA_ENV_NAME: [ "quartz-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      - HOST : [ "ruby" ]
-        CONDA_ENV_NAME: [ "ruby-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      # - HOST : [ "lassen" ]
-      #   ENVIRONMENT: [ "BATCH" ]
-      #   CONDA_ENV_NAME: [ "lassen-env" ]
-      #   RUNNER_TYPE: "batch"
-        # CONDA_ARCH: "ppc64le"
-  tags:
-    - $HOST
-    - $RUNNER_TYPE
   script:
     - mkdir -p ${CONDA_WORK_DIR}
     - test -d ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH} && exit 0
-    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-${CONDA_ARCH}.sh -O ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}.sh
-    - chmod +x ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}.sh
-    - ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}.sh -b -p ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}
+    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-${CONDA_ARCH}.sh -O ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}-${HOST}.sh
+    - chmod +x ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}-${HOST}.sh
+    - ${CONDA_WORK_DIR}/miniconda-${CONDA_ARCH}-${HOST}.sh -b -p ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}-${HOST}
     - *finalize
 
-conda-env-setup:
+.conda_env_setup:
   stage: setup
-  parallel:
-    matrix:
-      - HOST : [ "quartz" ]
-        CONDA_ENV_NAME: [ "quartz-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      - HOST : [ "ruby" ]
-        CONDA_ENV_NAME: [ "ruby-env" ]
-        RUNNER_TYPE: "shell"
-        CONDA_ARCH: "x86_64"
-      # - HOST : [ "lassen" ]
-      #   ENVIRONMENT: [ "BATCH" ]
-      #   CONDA_ENV_NAME: [ "lassen-env" ]
-      #   RUNNER_TYPE: "batch"
-        # CONDA_ARCH: "ppc64le"
-  tags:
-    - $HOST
-    - $RUNNER_TYPE
   script:
     - test -d ${CONDA_WORK_DIR}/${CONDA_ENV_NAME} && exit 0
-    - source ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}/etc/profile.d/conda.sh
+    - source ${CONDA_WORK_DIR}/miniconda3-${CONDA_ARCH}-${HOST}/etc/profile.d/conda.sh
     - eval "$(conda shell.posix hook)"
-    - conda update -y -n base -c defaults conda
+    - conda update -y -n base conda
     - conda create --prefix ${CONDA_WORK_DIR}/${CONDA_ENV_NAME} -y
+    - conda activate ${CONDA_WORK_DIR}/${CONDA_ENV_NAME}
```

Comment:
we needed this line only for mamba, thus since mamba is gone, this also can be removed

Reply:
removed in [commit](https://github.com/ExaWorks/SDK/pull/191/commits/788f978393e402d7ee8333da649bf24ea90a59e4)